### ISSUE TITLE
On initial connection to server. This error is thrown

### ISSUE THREAD

**Issue by Xorgadget** [Jackarunda/gmod#49]
[gmod] addons/gmod/lua/jmod/cl_init.lua:514: attempt to perform arithmetic on field 'targetBlindness' (a nil value)
  1. func - addons/gmod/lua/jmod/cl_init.lua:514
   2. unknown - lua/includes/extensions/net.lua:32

On initial connection to server. This error is thrown

**Comment by FreakingFission** [comment]
That was a bug from the implementation of the EZ Tear Gas Grenade, from what I can tell, that bug has been fixed, meaning you probably do not have the current version of JMOD.

**Comment by Xorgadget** [comment]
I pulled it fresh. Let me try again.

**Comment by Xorgadget** [comment]
A pull from master fixed the issue.
